### DISCUSSION TITLE
Github page not deploying!

### DISCUSSION THREAD

**Discussion by iyamk** [community/community#57529] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

How long do I have to wait until it's visible? I see a 404 error. I just chose the /docs folder and I want everything to work right away

**Comment by NyanKaungSet** [comment]
Hello @iyamk,

Normally, it takes 1-2 mins.  You can view it in `action` -> `pages build and deployment` workflow. 
Try deploying on `/(root)` folder. Make sure you have index.html in root.

> If you choose the docs folder on any branch as your publishing source, then later remove the `/docs` folder from that branch in your repository, your site won't build and you'll get a page build error message for a missing `/docs` folder. For more information, see "[Troubleshooting Jekyll build errors for GitHub Pages sites](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites#missing-docs-folder)."

Source: [GitHub Docs](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site)


It will be great if you provide the repo that you are trying to deploy. 

Best regards,
